crawler/utils/dockerutils.py

```diff
@@ -360,3 +363,31 @@ def get_docker_container_rootfs_path(long_id, inspect=None):
         raise DockerutilsException('Not supported docker storage driver.')
 
     return rootfs_path
+
+
+def poll_container_create_events(timeout=0.1):
+    try:
+        client = docker.Client(base_url='unix://var/run/docker.sock',
```

Comment:
This might be the reason why some events are lost. If it was the same client being reused, then the client would keep the list of events.